akka-distributed-data/src/main/resources/reference.conf

```diff
@@ -62,6 +62,10 @@ akka.cluster.distributed-data {
   delta-crdt {
     # enable or disable delta-CRDT replication
     enabled = on
+    
+    # Some complex deltas grow in size for each update and above this
+    # threshold such deltas are discarded and sent as full state instead.
+    max-delta-size = 200
```

Comment:
No, no, I think it is fine. I was just surprised that even such a seemingly large number can be exceeded.